app/src/main/java/com/example/keniphhomemaintenance/dwellings/DwellingsScreenView.kt

```diff
@@ -15,12 +23,12 @@ import com.example.keniphhomemaintenance.ui.cookbook.KeniphFloatingActionButton
 @Composable
 fun DwellingsScreenView(
     navController: NavController,
-    dwellingsViewModel: DwellingsViewModel = viewModel()
+    dwellingsViewModel: DwellingsViewModel = viewModel(
+        factory = DwellingsViewModelFactory(LocalContext.current)
+    )
 ) {
-
     val isDwellingDialogVisible = remember { mutableStateOf(false) }
-
-    val listOfDwellings = dwellingsViewModel.dwellingsList.value
+    val dwellingsList = dwellingsViewModel.dwellings.value
```

Comment:
Was messing around with this logic and changed the name by accident. Sorry Nick <3

Reply:
FYI that dwellingsList is the more common semantic approach